Changelog.md

```diff
@@ -9,6 +9,12 @@ Breaking Changes
 
 New
 
+* Routinator now keeps the last valid data from a publication point and
+  falls back to using that if an update to the publication point does not
+  have a valid manifest or the data does not match the manifest. ([#456]).
+* The new option `--fresh` causes Routinator to delete all locally stored
+  data before starting. This can be used when data corruption is
+  reported. ([#470])
```

Comment:
Perhaps refine this to say this can be used when _Routinator_ reports that its data is corrupted? Does Routinator actually report that?

Reply:
It does. And then exits.

I left out ‘Routinator’ there because of word repetition.